File: 10810/helloworld.py
#5 4
#1 2 3
#3 4 4
#1 4 1
#2 2 2
in_first = input("").split(" ")

first_input = int(in_first[0])
second_input = int(in_first[1])
count = 0
newArray = []

while first_input > count:
    newArray.append(0)
    count += 1


count2 = 0
while second_input > count2:
    range_input = input("")
    second_split = range_input.split(" ")
    [first , second , three] = second_split
    first_range = int(first)
    second_range = int(second)
    while first_range <= second_range:
        newArray[first_range - 1] = int(three)
        first_range += 1
    count2 += 1
    
print(" ".join(map(str, newArray)))

File: 10810/helloworld.c
#include <stdio.h>

int main(void)
{
    int firstIn, secondIn;
    scanf("%d %d",&firstIn , &secondIn);
    int newArray[firstIn];
    for (int i = 0; i < firstIn; i++)
    {
        newArray[i] = 0;
    }
    for (int j = 0; j < secondIn; j++)
    {
        int first , second, three;
        scanf("%d %d %d", &first , &second , &three);

        for (int k = first; k <= second; k++)
        {
            newArray[k - 1] = three;
        }
        
    }
    
    // Output
    for (int q = 0; q < firstIn; q++)
    {
        printf("%d ",newArray[q]);
    }
    
    
    return 0;
}
